mesonbuild/modules/rust.py

```diff
@@ -69,18 +72,25 @@ class FuncBindgen(TypedDict):
      KwargInfo('is_parallel', bool, default=False),
 ]
 
+def no_spaces_validator(arg: T.Optional[T.Union[str, T.List]]) -> T.Optional[str]:
+    if any((len(x.split()) > 1 for x in arg)):
```

Comment:
`x.split(maxsplit=1)`

Reply:
Although, I wonder if `if ' ' in x` might be simpler?

Reply:
I wanted to catch tabs as well. Probably I should have searched stackoverflow :) I'll see what I come up with.